### PR TITLE
Adds a wait and overwrites the labels for sriov

### LINKED ISSUE #67: Error deploying the sriov operator
```
error: unable to recognize “STDIN”: no matches for kind “SriovNetworkNodePolicy” in version “sriovnetwork.openshift.io/v1”
```

### PR DESCRIPTION
**Pull request by e-minguez**
# Description

Currently the operator is deployed and then configured... but in order to be successfully deployed it needs to wait until the operator is fully created, otherwise it fails with the following issue:

Fixes #67

I've also added a 'force' label because otherwise if the node is labeled, it won't be overwritten.

## Type of change

Please select the appropriate options:

- [X] Bug fix (non-breaking change which fixes an issue)

## Testing

- [x] Deployed the sriov-operator with the changes

**Test Configuration**:

- Versions: 

```
Client Version: 4.3.0-0.nightly-2019-12-04-031453
Server Version: 4.3.0-0.nightly-2019-12-04-031453
Kubernetes Version: v1.16.2
openshift-sriov-network-operator   sriov-network-operator-subscription   sriov-network-operator   redhat-operators   alpha
```
- Hardware: Baremetal environment

## Checklist

- [x] I have performed a self-review of my own code
- [x] If the change is a fix, it should have an issue. The review should make sure the comments state the issue (not just the number) and it should use the keywords that will close the issue on merge.
- [x] A change should not being merged unless it passes CI or there is a comment/update saying what testing was passed.
- [x] PRs should not be merged unless positively reviewed.

### CODE REVIEW

File: features/sriov/deploy.sh [hunk 1]
```diff
@@ -55,6 +55,14 @@ daemonset(){
   envsubst '\${nic} \${NODESELECTOR}' < ./10-sriov-daemonset.yaml | oc apply -f - > /dev/null || die "Error creating ds"
 }
 
+wait_for_ready(){
+  # This is to prevent creating the policy before the operator has been deployed like:
+  # error: unable to recognize “STDIN”: no matches for kind "SriovNetworkNodePolicy" in version "sriovnetwork.openshift.io/v1"
+  info "Waiting for the operator to be ready..."
+  # shellcheck disable=SC2154
+  while ! oc wait --for condition=ready pods -l name=sriov-network-operator -n "${operatornamespace}" --timeout=2400s; do sleep 10 ; done
```
Comment (cynepco3hahue) [inline features/sriov/deploy.sh]: do we really need to wait 40 minutes for the operator to be ready

Comment (e-minguez) [reply features/sriov/deploy.sh]: It was a random choice. Any suggestions? 5 minutes would be ok?

Comment (cynepco3hahue) [reply features/sriov/deploy.sh]: if it will be enough works for me:)

Comment (e-minguez) [reply features/sriov/deploy.sh]: I've set 300 by default but it can be tweaked if needed.

**Pull request merged**